src/misc/article.v2.yaml

```diff
@@ -0,0 +1,78 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Article
+type: object
+allOf:
+  - $ref: ../snippets/article.v1.yaml
+  - properties:
+        issue:
+            type: integer
+            minimum: 1
+        copyright:
+            allOf:
+              - type: object
+                properties:
+                    statement:
+                        type: string
+                        minLength: 1
+                required:
+                  - statement
+              - oneOf:
+                  - properties:
+                        license:
+                            type: string
+                            enum:
+                              - CC0-1.0
+                    required:
+                      - license
+                  - properties:
+                        license:
+                            type: string
+                            enum:
+                              - CC-BY-1.0
+                              - CC-BY-2.0
+                              - CC-BY-2.5
+                              - CC-BY-3.0
+                              - CC-BY-4.0
+                        holder:
+                            type: string
+                            minLength: 1
+                    required:
+                      - license
+                      - holder
+        authors:
+            type: array
+            items:
+                $ref: author.v1.yaml
+            minItems: 1
+        reviewers:
+            type: array
+            items:
+                $ref: reviewer.v1.yaml
+        dataSets:
+            type: object
+            properties:
+                generated:
+                    type: array
+                    items:
+                        $ref: ../misc/data-set.v1.yaml
+                used:
+                    type: array
+                    items:
+                        $ref: ../misc/data-set.v1.yaml
+        ethics:
+            type: array
+            items:
+                $ref: ../blocks/paragraph.v1.yaml
+            minItems: 1
+        funding:
+            $ref: ../misc/funding.v1.yaml
+        additionalFiles:
+            title: Additional files
+            type: array
+            items:
+                $ref: ../misc/asset-file.v1.yaml
```

Comment:
Could you please take a look here and confirm - perhaps by @thewilkybarkid - should this be ``asset-file.v1.yaml`` or should it be ``asset-file.v2.yaml``?

I'm testing the schema generating an altered article 26759, except it is not passing validation because the transparent reporting form in the additionalFiles has a label and not a title.

Also above in ``article.v1.yaml`` it is referencing to use ``asset-file.v2.yaml``.

I wanted to confirm these before I presumably change how the title or label is set depending on whether it is a figure sourceData or an article additionalFiles item (which I guess in the XML parser goes through the same function).

Reply:
Good spot, I've just found a bunch of these. :doh: